<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectModeWidgetUI</class>
 <widget class="QWidget" name="SelectModeWidgetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtWidgetsClass</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pb_selectall">
     <property name="text">
      <string>Select All</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pb_unselectall">
     <property name="text">
      <string>Unselect All</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="cb_solid">
     <property name="text">
      <string>Solid</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="cb_shell">
     <property name="text">
      <string>Shell</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="cb_face">
     <property name="text">
      <string>Face</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="cb_wire">
     <property name="text">
      <string>Wire</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="cb_edge">
     <property name="text">
      <string>Edge</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="cb_vertax">
     <property name="text">
      <string>Vertax</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="cb_shape">
     <property name="text">
      <string>Shape</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
